src/arearange/area-range-component.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
+import * as Highcharts from "highcharts";
+import more from "highcharts/highcharts-more";
+import HighchartsReact from "highcharts-react-official";
+import { getRangeData } from "../service";
+import { AreaRangeData } from "./area-range-data";
+import { toAreaRangeSeries } from "../util";
+more(Highcharts);
+
+interface AreaRangeProps {
+    type: string;
+    stateCode: string;
+}
+
+interface AreaRangeState {
+    optionsForAllCharts: Highcharts.Options[];
+}
+
+const optionsDelegate = (rangeData: AreaRangeData): Highcharts.Options => {
+    return {
+        title: {
```

Comment:
currently some of the chart setting are in backend some are in frontend. Let's move charting related settings to frontend only.